MoTrPAC/motrpac-frontend
# Connects to #120. Related Studies and HERITAGE Proteomics.

### Key Changes:

1. Added new Related Studies page.
2. Added new HERITAGE Proteomics page.

### Technical Notes:

1. Prior to reviewing/testing the branch, run `nvm use v14.15.4`.
2. Also, run `rm -fr node_modules/` and `rm -f yarn.lock`. Then run `yarn install`.

### Steps to test:

1. Click on the **Related Studies** link in the top navbar and expect to be routed to the new **Related Studies** page.
2. Click on the **HERITAGE Proteomics** link on the page and expect to be routed to the new **HERITAGE Proteomics** page.
3. Click on the **Download HERITAGE Proteomics dataset** button and expect to see the download to be completed successfully.
4. Click on the **Back to Related Studies** link and expect to be routed back to the **Related Studies** page.
